src/main/resources/application.yaml

```diff
@@ -355,14 +355,155 @@ darts:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      close-old-cases:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      close-old-unfinished-transcriptions:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      daily-list-housekeeping:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      dets-to-arm:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      external-datastore-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      generate-case-document:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT45M
+      generate-case-document-for-retention-date:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      inbound-transcription-annotation-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
       inbound-audio-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
         unstructured-minimum-duration: 24h
         default-batch-size: 100000
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
       inbound-to-unstructured:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
         threads: 20
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      manual-deletion:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      outbound-audio-deleter:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT40M
+      process-arm-response-files:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT1H30M
+      process-arm-rpo-pending:
+        system-user-email: ${darts.automated.task.common-config.system-user-email:}
+        lock:
+          at-least-for: PT1M
+          at-most-for: PT5H
```

Comment:
The LLD has this as 60 mins

Reply:
I dont see a job in the LLD for ProcessARMRPOPending but have updated this to be 90m

Reply:
It's called `ProcessE2EArmRpoPending` in the LLD - I assume it's the same task @hemantasharma1129 ? We should probably update the LLD to align as i've noticed a few inconsistencies.

Reply:
I have added the missing job 'ProcessARMRPOPending' in the LLD. The implementation was done under DMP-3529.